pack/tic/itd.json

```diff
@@ -1,4 +1,56 @@
 [
+	{
+		"code": "07108",
+		"cost": 2,
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"flavor": "\"If you blow that thing one more time, I'm gonna make you eat it.\"",
+		"illustrator": "Robert Laskey",
+		"name": "Riot Whistle",
+		"pack_code": "itd",
+		"position": 108,
+		"quantity": 2,
+		"skill_willpower": 1,
+		"slot": "Accessory",
+		"text": "You may take an additional action during your turn, which can only be used to <b>engage</b>.",
+	    "traits": "Item. Tool.",
+		"type_code": "asset",
+	    "xp": 0
+	},
+	{
+		"code": "07109",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"flavor": "Lita narrowed her eyes and tightened her grip. They wouldn't go away. Not this time. Not ever again.",
+		"illustrator": "Alexandr Elichev",
+		"name": "Righteous Hunt",
+		"pack_code": "itd",
+		"position": 109,
+		"quantity": 2,
+		"skill_agility": 1,
+		"skill_combat": 1,
+		"text": "<b>Engage.</b> Choose an enemy up to 2 locations away. Move (one location at a time) to that enemy's location, engage it, and add [bless] tokens to the chaos bag equal to that enemy's horror value.",
+		"traits": "Tactic. Blessed.",
+		"type_code": "event",
+		"xp": 1
+	},
+	{
+		"code": "07110",
+		"cost": null,
+		"deck_limit": 1,
+		"faction_code": "guardian",
+		"illustrator": "Tiziano Baracchi",
+		"name": "Sacred Covenant",
+		"pack_code": "itd",
+		"permanent": true,
+		"position": 110,
+		"quantity": 1,
```

Comment:
Are you confident on the quantities of the covenants? I haven't got my pack yet to check...

Reply:
Yeah, all reports are there are only 1 of each (hence why we get 15 player cards instead of the usual 10)